models/blogModel.js

```diff
@@ -24,7 +24,7 @@ const blogSchema = new mongoose.Schema(
                     type: mongoose.Schema.Types.ObjectId,
                     ref: "User",
                 },
-                content: String,
+                type: String,
```

Comment:
what does this type signifies?

Reply:
type signifies what the type of comment is

Reply:
then where are we storing content of the comment?